lib/perl/Genome/VariantReporting/Suite/Joinx/Dbsnp/DbsnpInterpreter.pm

```diff
@@ -44,16 +44,18 @@ sub _interpret_entry {
     my %return_values;
 
     for my $variant_allele (@$passed_alt_alleles) {
-        if (!defined $caf) {
-            $return_values{$variant_allele}->{allele_frequency} = undef;
-        }
-        elsif (defined $caf->{$variant_allele}) {
-            $return_values{$variant_allele}->{allele_frequency} = $caf->{$variant_allele};
-        }
-        else {
-            die sprintf("No allele frequency for allele ($variant_allele) given in caf: %s",
-                Data::Dumper::Dumper($caf));
+        if (defined($entry->info('CAF')) && !defined($caf->{$variant_allele})) {
```

Comment:
Also, would it be better to use `$parser->TAG` instead of hard-coding `'CAF'` here?